### PR TITLE
Fix cfg directives for frida-asan

### PR DESCRIPTION
**Pull request by s1341**
Fixes missing/broken cfg directives from @tokatoka's frida-asan-x86_64 PR.

### CODE REVIEW

File: libafl_frida/src/helper.rs [hunk 1]
```diff
@@ -836,6 +836,7 @@ impl<'a> FridaInstrumentationHelper<'a> {
         ));
     }
 
+    #[cfg(target_arch = "x86_64")]
```
Comment (tokatoka) [inline libafl_frida/src/helper.rs:839]: i have fixed this :)

Comment (tokatoka) [reply libafl_frida/src/helper.rs:839]: at line 842

Comment (s1341) [reply libafl_frida/src/helper.rs:839]: I must have missed it... Thanks!

Comment (domenukk) [reply libafl_frida/src/helper.rs:839]: so can I merge this pr @tokatoka or should we fix something?

Comment (domenukk) [reply libafl_frida/src/helper.rs:839]: Ah guess we can drop this line?

Comment (s1341) [reply libafl_frida/src/helper.rs:839]: Did it get duplicated?

Comment (domenukk) [reply libafl_frida/src/helper.rs:839]: Yes. Fixing it right now

Comment (s1341) [reply libafl_frida/src/helper.rs]: Thanks. Sorry.

**Pull request merged**